### PR TITLE
adds less than more!

### PR DESCRIPTION
**Pull request by longhat-plenty**

### CODE REVIEW

File: src/css/swag.scss [hunk 1]
```diff
@@ -172,8 +202,56 @@ $prop-format: var(--plentyRedSeaPink);
...
+        padding-left: 0.5rem;
+      }
+      .authorization__btn {
+        order: 2;
+      }
+      .opblock-summary-path {
+        flex: none;
+        min-width: 100%;
+        background: rgba(0, 0, 0, 0.2);
+        margin: 0.25rem 0 0.25rem 0;
```
Comment (jboewerplenty) [inline src/css/swag.scss:232]:
```suggestion
        margin: 0.25rem 0;
```

File: src/css/swagger-ui.css [hunk 2]
```diff
@@ -1,5 +1,10 @@
+#swagger-ui {
+  margin-bottom: 4em;
+}
+
 .information-container.wrapper,
-.float-right {
+.float-right,
+section.models {
   display: none !important;
```
Comment (jboewerplenty) [inline src/css/swagger-ui.css:8]: Is there a way of getting rid of this display:none?

File: src/css/swagger-ui.css [hunk 3]
```diff
@@ -115,8 +139,10 @@
 .swagger-ui .opblock-summary-description a:focus {
   color: var(--color-white);
 }
-.swagger-ui .swagger-container .wrapper {
-  display: none;
+.swagger-ui .opblock .opblock-summary-method {
+  font-size: 1rem;
+  min-width: 6em;
+  padding: 0.25em 0.25em;
```
Comment (jboewerplenty) [inline src/css/swagger-ui.css:145]:
```suggestion
  padding: 0.25em;
```

File: src/css/swagger-ui.css [hunk 4]
```diff
@@ -126,6 +152,49 @@
...
+    padding-left: 0.5rem;
+  }
+  .swagger-ui .opblock .authorization__btn {
+    order: 2;
+  }
+  .swagger-ui .opblock .opblock-summary-path {
+    flex: none;
+    min-width: 100%;
+    background: rgba(0, 0, 0, 0.2);
+    margin: 0.25rem 0 0.25rem 0;
```
Comment (jboewerplenty) [inline src/css/swagger-ui.css:180]:
```suggestion
    margin: 0.25rem 0;
```

**Review by jboewerplenty — CHANGES_REQUESTED** [review]

File: src/css/swagger-ui.css (hunk 2 above)
Comment (longhat-plenty) [reply src/css/swagger-ui.css:8]: what do you mean exactly? Swagger comes with generated portions we just dont want to show. Digging into swagger for getting portions not rendered at all kinda sounds too much of effort for simply using display: none;

**Review by jboewerplenty — APPROVED** [review]

**Pull request merged**